driver/pace/driver/driver.py

```diff
@@ -595,9 +595,9 @@ def _critical_path_step_all(
                     self.end_of_step_update(
                         dycore_state=self.state.dycore_state,
                         phy_state=self.state.physics_state,
-                        u_dt=self.state.tendency_state.u_dt.storage,
-                        v_dt=self.state.tendency_state.v_dt.storage,
-                        pt_dt=self.state.tendency_state.pt_dt.storage,
+                        u_dt=self.state.tendency_state.u_dt.data,
+                        v_dt=self.state.tendency_state.v_dt.data,
+                        pt_dt=self.state.tendency_state.pt_dt.data,
                         dt=float(dt),
```

Comment:
Quantity should work here.
```suggestion
                        u_dt=self.state.tendency_state.u_dt,
                        v_dt=self.state.tendency_state.v_dt,
                        pt_dt=self.state.tendency_state.pt_dt,
                        dt=float(dt),
```